tests/test_e2e_10_mef_eline.py

```diff
@@ -613,3 +693,649 @@ def test_080_create_and_remove_ten_circuits_ten_times(self):
+
+        # It sets a new circuit's priority
+        requests.patch(api_url + evc1, data=json.dumps(payload),
+                       headers={'Content-type': 'application/json'})
+
+        time.sleep(10)
+
+        # It verifies EVC's data
+        response = requests.get(api_url + evc1)
+        data = response.json()
+        assert data['priority'] == priority
+
+    def test_125_patch_queue_id(self):
+
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        evc1 = self.create_evc(100)
+
+        queue_id = 100
+        payload = {
+            "queue_id": queue_id
+        }
+
+        # It sets a new circuit's queue_id
+        requests.patch(api_url + evc1, data=json.dumps(payload),
+                       headers={'Content-type': 'application/json'})
+
+        time.sleep(10)
+
+        # It verifies EVC's data
+        response = requests.get(api_url + evc1)
+        data = response.json()
+        assert data['queue_id'] == queue_id
+
+    def test_130_patch_dynamic_backup_path(self):
+
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        evc1 = self.create_evc(100)
+
+        dynamic_backup_path = True
+        payload = {
+            "dynamic_backup_path": dynamic_backup_path
+        }
+
+        # It sets a new circuit's dynamic_backup_path
+        requests.patch(api_url + evc1, data=json.dumps(payload),
+                       headers={'Content-type': 'application/json'})
+
+        time.sleep(10)
+
+        # It verifies EVC's data
+        response = requests.get(api_url + evc1)
+        data = response.json()
+        assert data['dynamic_backup_path'] == dynamic_backup_path
+
+    """The EVC is returning active=False"""
+    @pytest.mark.xfail
+    def test_135_patch_primary_path(self):
+
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        payload1 = {
+            "name": "my evc1",
+            "enabled": True,
+            "uni_a": {
+                "interface_id": "00:00:00:00:00:00:00:01:1",
+            },
+            "uni_z": {
+                "interface_id": "00:00:00:00:00:00:00:02:1"
+            },
+            "primary_path": [
+                {"endpoint_a": {"id": "00:00:00:00:00:00:00:01:3"},
+                 "endpoint_b": {"id": "00:00:00:00:00:00:00:02:3"}}
+            ]
+        }
+        response = requests.post(api_url, data=json.dumps(payload1),
+                                 headers={'Content-type': 'application/json'})
+        data = response.json()
+        evc1 = data['circuit_id']
+        time.sleep(10)
+        payload2 = {
+            "primary_path": [
+                {"endpoint_a": {"id": "00:00:00:00:00:00:00:01:4"},
+                 "endpoint_b": {"id": "00:00:00:00:00:00:00:03:3"}}
+            ]
+        }
+        # It sets a new circuit's primary_path
+        response = requests.patch(api_url + evc1, data=json.dumps(payload2),
+                                  headers={'Content-type': 'application/json'})
+        assert response.status_code == 200
```

Comment:
this should return 400!

Reply:
Resolved with commit [860897d](https://github.com/amlight/kytos-end-to-end-tests/pull/69/commits/860897dbefb1db7e0be42a533a8f748f7efa2535)